doc-Deployment_Planning_Guide/topics/planning.adoc

```diff
@@ -64,7 +64,13 @@ With centralized administration, the remote_queue_put leverages a new system-to-
 The operations initiated from the global region are subject to the role-based access control (RBAC) rules on the remote region. The user in the remote region which matches the logged-in user's *user ID* will be used to enforce RBAC in the target region. The operation will fail on the remote system if the user does not have the required permissions. 
 ====
 
-For information on how to configure centralized administration via database replication, see the section on _Configuring Database Replication and Centralized Administration_ in the https://access.redhat.com/documentation/en/red-hat-cloudforms/4.5/single/general-configuration/[General Configuration] guide.
+ifdef::cfme[]
+In CloudForms 4.5 and above, configuring database replication automatically enables centralized configuration and no further configuration is needed. See the section on _Configuring Database Replication and Centralized Administration_ in the https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/general_configuration/#configuring_database_replication[General Configuration] guide.
```

Comment:
Hey Suyog,
Here I might re-word "and no further configuration is needed" to "eliminating the need for further configuration" just to avoid using the passive voice at the end.